listview/.editorconfig

```diff
@@ -0,0 +1,15 @@
+root = true
+
+[*]
```

Comment:
Should we commit this file ?

Reply:
This is an autogenerated file that sets a "common" style when editing the demo-angular via VS Code. I think we should leave those files committed to enforce the same code style and formatting.